include/picongpu/plugins/openPMD/openPMDWriter.hpp

```diff
@@ -1223,6 +1363,9 @@ Make sure that environment variable OPENPMD_BP_BACKEND is not set to ADIOS1.
                 }
                 log<picLog::INPUT_OUTPUT>("openPMD: ( end ) writing particle species.");
 
+                log<picLog::INPUT_OUTPUT>("openPMD: ( begin ) writing RNG states.");
```

Comment:
This code should be guarded by

```C++
if(mThreadParams.isCheckpoint == true)
{
// ...
}
```

RNG stated should not be stored into normal data dumps.

Reply:
True. Why don't we do it for the id provider just below?

Reply:
It is a bug if we do not do it for the id provider too.

Reply:
Okay, fixed only in RNG. Id Provider will be fixed in another PR